admin-dev/themes/new-theme/js/app/pages/permission/components/row.vue

```diff
@@ -0,0 +1,268 @@
+<template>
+  <div>
+    <div :class="{parent, 'bg-light': levelDepth === 1}" class="row permission-row">
+      <div class="col-lg-4">
+        <template v-for="i in levelDepth" v-if="i > 2">
```

Comment:
Always use key with v-for. https://vuejs.org/v2/style-guide/

Reply:
Not when it's not an object.